api/src/modules/projects/dto/create.project.dto.ts

```diff
@@ -54,18 +56,20 @@ export class CreateProjectDTO {
   @IsEnum(Object.values(PlanningUnitGridShape))
   planningUnitGridShape?: PlanningUnitGridShape;
 
-  @ApiProperty()
+  @ApiPropertyOptional()
   @IsOptional()
   @IsNumber()
   planningUnitAreakm2?: number;
 
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({
+    description: `Geometry part of GeoJson; MultiPolygon or Polygon`,
+  })
   @IsOptional()
   @IsObject()
-  extent?: Record<string, unknown>;
+  extent?: MultiPolygon | Polygon;
 
   @ApiPropertyOptional()
   @IsOptional()
   @IsObject()
-  metadata?: Record<string, unknown>;
+  metadata?: Record<string, string>;
```

Comment:
I think this could stay as `Record<string, unknown>` for the moment (we're not using it IIRC and if we do start using it we may want to have more precise typing for this prop's values, but just in case, I'd leave this open for the time being).

Reply:
```suggestion
  metadata?: Record<string, unknown>;
```